cobbler/actions/reposync.py

```diff
@@ -570,12 +573,16 @@ def yum_sync(self, repo):
             proxy = repo.proxy
         (cert, verify) = self.gen_urlgrab_ssl_opts(repo.yumopts)
 
-        # FIXME: These two variables were deleted
-        repodata_path = ""
-        repomd_path = ""
+        repodata_path = os.path.join(temp_path, "repodata")
+        repomd_path = os.path.join(repodata_path, "repomd.xml")
         if os.path.exists(repodata_path) and not os.path.isfile(repomd_path):
             shutil.rmtree(repodata_path, ignore_errors=False, onerror=None)
 
+        repodata_path = os.path.join(temp_path, "repodata")
+        if os.path.exists(repodata_path):
+            self.logger.info("Deleted old repo metadata for %s" % repodata_path)
```

Comment:
```suggestion
            self.logger.info("Deleted old repo metadata for %s", repodata_path)
```

This is a logger and thus no percent formatting. See here: https://docs.python.org/3/howto/logging.html#logging-variable-data

Reply:
I made the requested changes.

Reply:
Ok... This is actually failing for me now:
```
received on stderr: 
Exception occurred: <class 'TypeError'>
Exception value: info() takes 2 positional arguments but 3 were given
Exception Info:
  File "/usr/lib/python3.6/site-packages/cobbler/actions/reposync.py", line 161, in run
    self.sync(repo)

  File "/usr/lib/python3.6/site-packages/cobbler/actions/reposync.py", line 204, in sync
    self.yum_sync(repo)

  File "/usr/lib/python3.6/site-packages/cobbler/actions/reposync.py", line 580, in yum_sync
    self.logger.info("Deleted old repo metadata for %s", repodata_path)
```

I'm gonna work through and resubmit a fix for this.

Reply:
Oh damn. That could be my error. Is 3.2.x still using the custom `clogger.py` implementation? If yes then revert the applied change since my change will only work if the default Python logging module is used.

Reply:
Yes, sadly, 3.2.x is still using the `clogger.py` implementation.  I will revert this change and push.